strings/json_lib.c

```diff
@@ -1562,15 +1562,7 @@ int json_unescape(CHARSET_INFO *json_cs,
     }
     if (c_len == MY_CS_ILUNI)
     {
-      /*
-        Result charset doesn't support the json's character.
-        Let's replace it with the '?' symbol.
-      */
-      if ((c_len= my_ci_wc_mb(res_cs, '?', res, res_end)) > 0)
-      {
-        res+= c_len;
-        continue;
-      }
+      return -1;
```

Comment:
it looks like there **is** a test case for that, but, please, confirm

Reply:
nope, there doesn't appear to be, at least in 10.11. based on above, I'm not sure it reachable.